data/timeline.service.in

```diff
@@ -5,5 +5,5 @@ Documentation=man:snapper(8) man:snapper-configs(5)
 
 [Service]
 Type=simple
-ExecStart=/usr/lib/snapper/systemd-helper --timeline
+ExecStart=@libexecdir@/snapper/systemd-helper --timeline
```

Comment:
It should be fixed in the spec with the passing of `--libexecdir=%{_libexecdir}` at configure time. In Mageia, this works fine because `%configure` already passes that in.